opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -285,12 +281,11 @@ namespace Opm {
             std::vector<double> residual_norms;
             perfTimer.reset();
             perfTimer.start();
-            report.converged = getConvergence(timer, iteration,residual_norms);
+            // the step is not considered converged until at least minIter iterations is done
```

Comment:
Is this change necessary? I'd like the report to state clearly that "yes, I converged" in this case, making it possible to tell the difference between the reasons for taking one more iteration (below min iters or not converged (== too large residuals in my mind)).